### DISCUSSION TITLE
Github dashboard guide

### DISCUSSION THREAD

**Discussion by d-ariho** [Azure/CCOInsights#306] [Q&A] [answered]
Hi,
Just wanted to know if a new github dashboard guide was going to be created? The old one was removed over a month ago. 
Thanks

**Comment by josunefon** [comment, marked as answer]
Hi @d-ariho 

We moved all the documentation to the [wiki](https://github.com/Azure/CCOInsights/wiki/GitHub%20Dashboard%20-%20Deployment%20Guide)
